Cargo.lock

```diff
@@ -4829,7 +4823,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "07033963ba89ebaf1584d767badaa2e8fcec21aedea6b8c0346d487d49c28667"
 dependencies = [
  "cfg-if",
- "windows-targets 0.52.6",
+ "windows-targets 0.48.5",
```

Comment:
Do we know why this crate is moving to a lower version?

Reply:
Cargo has issues with automatically upgrading/downgrading windows dependencies for some reason. I do not know the exact reason for it.

Reply:
Running ` cargo update --package libloading` undoes the downgrade. The windows build still passes on my [fork](https://github.com/mukilan/servo/actions/runs/16244853580/job/45866175760), so I've pushed [fd05a26](https://github.com/servo/servo/pull/38013/commits/fd05a263f0e7732d74f294f8dba7f24a34cf722e) to this branch.